### ISSUE TITLE
Accessibility Check doesn't identify thead

### ISSUE THREAD

**Issue by mbennett417** [joomla/joomla-cms#42678]
### Steps to reproduce the issue
In Joomla 5.0.2 when creating an article containing a table, run the Checker and it will give an Accessibility Error on the table because it lacks a header. The table has a correct header line that displays correctly.

### Expected result
This should not receive and Accessibility error.

### Actual result
I expect it works correctly for users with screen readers, but it's incorrect for the developer.

### System information (as much as possible)
Joomla 5.0.2 (including previous 5.x versions)
PHP 8.1.27
Database 10.6.16-MariaDB
Litespeed webserver
Hosted on Rochen

### Additional comments
I've tried this on different sites and have gotten the same results every time. It just doesn't seem to "see" the thead tag.

**Comment by brianteeman** [comment]
Please post the html code for the table you are creating.

**Comment by mbennett417** [comment]
Kind of a monster but here is is.
[FAQ.txt](https://github.com/joomla/joomla-cms/files/13994210/FAQ.txt)

**Comment by brianteeman** [comment]
The accessibility checker is correct. You do not have valid table headers
You have

```
	<thead>
		<tr class="h3" style="height: 50px; text-align: center;">
			<td style="text-align: center;"><span style="font-size: 18pt;"><strong>Question</strong></span></td>
			<td style="text-align: center;"><span style="font-size: 18pt;"><strong>Answer</strong></span></td>
		</tr>
	</thead>
```

Correct code would be

```
	<thead>
		<tr class="h3" style="height: 50px; text-align: center;">
			<th style="text-align: center;"><span style="font-size: 18pt;"><strong>Question</strong></span></th>
			<th style="text-align: center;"><span style="font-size: 18pt;"><strong>Answer</strong></span></th>
		</tr>
	</thead>
```

**Comment by mbennett417** [comment]
OK, thanks. I didn't understand (obviously) the role that th plays.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42678">issues.joomla.org/tracker/joomla-cms/42678</a>.</sub>

**Comment by Quy** [comment]
Closing as no a core issue. Thanks Brian!

**Comment by brianteeman** [comment]
It's awesome that you are testing and trying to be accessible
